addons/portal/views/portal_templates.xml

```diff
@@ -35,6 +35,21 @@
                 </div>
             </div>
         </xpath>
+        <xpath expr="//head/meta" position="after">
```

Comment:
Note: we discussed in discord to support it directly in the portal templates. Definitively better that way, thanks for update :)